tree/ntuple/v7/inc/ROOT/RNTupleWriter.hxx

```diff
@@ -108,23 +108,27 @@ public:
    /// \return The number of uncompressed bytes written.
    std::size_t Fill(REntry &entry) { return fFillContext.Fill(entry); }
    /// Fill an entry into this ntuple, but don't commit the cluster. The calling code must pass an RNTupleFillStatus
-   /// and check RNTupleFillStatus::ShouldCommitCluster.
-   void FillNoCommit(REntry &entry, RNTupleFillStatus &status) { fFillContext.FillNoCommit(entry, status); }
+   /// and check RNTupleFillStatus::ShouldFlushCluster.
+   void FillNoFlush(REntry &entry, RNTupleFillStatus &status) { fFillContext.FillNoFlush(entry, status); }
    /// Flush column data, preparing for CommitCluster or to reduce memory usage. This will trigger compression of pages,
    /// but not actually write to storage (unless buffered writing is turned off).
    void FlushColumns() { fFillContext.FlushColumns(); }
+   /// Flush so far filled entries to storage
+   void FlushCluster() { fFillContext.FlushCluster(); }
    /// Ensure that the data from the so far seen Fill calls has been written to storage
    void CommitCluster(bool commitClusterGroup = false)
    {
-      fFillContext.CommitCluster();
+      fFillContext.FlushCluster();
       if (commitClusterGroup)
          CommitClusterGroup();
    }
 
    std::unique_ptr<REntry> CreateEntry() { return fFillContext.CreateEntry(); }
 
+   /// Return the entry number that was last flushed in a cluster.
+   NTupleSize_t GetLastFlushed() const { return fFillContext.GetLastFlushed(); }
    /// Return the entry number that was last committed in a cluster.
-   NTupleSize_t GetLastCommitted() const { return fFillContext.GetLastCommitted(); }
+   NTupleSize_t GetLastCommitted() const { return fFillContext.GetLastFlushed(); }
```

Comment:
In principle no, but for the sequential writer a flush implies a commit, so left the original `CommitCluster` interface in place and then I thought it would be nice to have a matching `GetLastCommitted()` function. We can remove it at any point though.